pkg/controller/virtinformers.go

```diff
@@ -543,10 +543,43 @@ func (f *kubeInformerFactory) VirtualMachine() cache.SharedIndexInformer {
 	})
 }
 
+func GetVirtualMachineExportInformerIndexers() cache.Indexers {
```

Comment:
I don't see either of these indexes exercised in unit tests

Reply:
Added unit tests for this.